libraries/AP_Motors/AP_MotorsHeli_RSC.h

```diff
@@ -113,3 +152,47 @@ class AP_MotorsHeli_RSC {
     // calculate_desired_throttle - uses throttle curve and collective input to determine throttle setting
     float           calculate_desired_throttle(float collective_in);
 };
+
+class RSCThrCrvInt16Param {
+public:
+    RSCThrCrvInt16Param(void);
+
+    static const struct AP_Param::GroupInfo var_info[];
+
+    void set_thrcrv_enable(int8_t setenable) {enable = setenable; }
+    float * get_thrcrv() {
```

Comment:
this is a bit odd ... seems to be doing a cast via a static array

Reply:
Now I see what you were referring to. It was confusing at first, as this part of the code had not been changed in this PR.